### PR TITLE
Added EasyPost localization keys for errors 

### PR DESCRIPTION
**Pull request by pgmichael**
https://dudamobile.atlassian.net/browse/SNIPCORE-10?atlOrigin=eyJpIjoiZjJlODJkYzY5ZTRiNDAyMmI1MTM5NDNlY2ZjZjhkMDciLCJwIjoiaiJ9

### CODE REVIEW

**Review by chuckcote — APPROVED** [review]
Looks good :)

File: locales/en.json [hunk 1]
```diff
@@ -333,7 +334,33 @@
             "no_rates_found": {
                 "title": "No shipping methods are available for your order",
                 "description": "Please review your shipping address or contact us."
-            }
+            },
+            "E.COUNTRY.UNSUPPORTED": "This country is not supported in this mode. Try in production mode.",
```
Comment (dominic-blain) [inline locales/en.json:338]: I'm wondering if we want to front all of these error messages as-is to the user. For simple error message with clear actionnables for the user (ie: House number is missing), we could relay the message but for everything that is not in the user's control like "We can not provide enough data in this country to satisfy this request. Please try again later.", we could show the generic error message asking them to confirm their address.

What do you think?

Comment (pgmichael) [reply locales/en.json:338]: Yes, we could tweak some of the messages to be more useful to the end user! For instance, this message in particular occurs, it probably means the merchant inverted his `Test` and `Live` API key. We could add something like `contact us for more details` so the customer contacts the merchant about the issue.

Feel free to add some suggestions for the lines you don't find particularly useful.

**Review by dominic-blain — APPROVED** [review]
Small typo fix, otherwise your changes are 💯

Note: I wonder what the hell is a "magnet street syndrome" 😂

**Pull request merged**